src/lib/workspace/Subsetting/hooks.ts

```diff
@@ -58,6 +59,7 @@ export const processGridData = (
       return {
         Header: `${entity.displayName} ID`,
         accessor: `${entity.id}/${columnID}`,
+        disableSortBy: true,
```

Comment:
curious who uses disableSortBy? Clearly someone does since it works lol i just couldn't find it.

Reply:
Good question! It's passed through to the `DataGrid` prop `columns`, whose type is an array of `Column`s, a type that's imported straight from `react-table` in CoreUI's `DataGrid/index.tsx`. I realize this means the full API is opaque to the CoreUI user, so I added a link to the `react-table` docs in the `columns` prop description.

Reply:
ahh that's great! Thank you!